pycheribuild/utils.py

```diff
@@ -414,7 +414,10 @@ def __parse_etc_os_release() -> "dict[str, str]":
         with Path("/etc/os-release").open(encoding="utf-8") as f:
             d = {}
             for line in f:
-                k, v = line.rstrip().split("=", maxsplit=1)
+                line = line.strip()
```

Comment:
I don't know why it was just an rstrip before but that shouldn't need changing for this?

Reply:
With only rstrip comments with preceding space won't be ignored, although I doubt that will ever come up in the wild.
In addition, using rstrip will have the key include any prefixing whitespace if there is any, which isn't desirable? I can't think of a reason it would be?

Reply:
I think the only reason it was rstrip before is that I assumed that was sufficient. I agree it should be changed to strip.

Reply:
And it's probably not worth splitting this out into a separate commit